Lost Kingdoms/UniversalDIT_Flat_Pressed.json

```diff
@@ -2114,7 +2114,7 @@
       "THUMBR": "../#DefaultDevices/DualShock4/Flat_Pressed/R3.png"
     }
   },
-  "generated_folder_name": "#Generated_",
+  "Generated_LostKingdoms": "#Generated_",
```

Comment:
The line must be like this.
`  "generated_folder_name": "#Generated_LostKingdoms",`